src/content/docs/en/reference/container-reference.mdx

```diff
@@ -64,6 +84,23 @@ const container = await experimental_AstroContainer.create({
 const result = await container.renderToString(ReactWrapper);
 ```
 
+##### Adding a renderer manually
+
+When the container is called at runtime, or inside other "shells", the `astro:container` virtual module's helper functions are not available. Import the necessary server renderer manually and store it inside the container using `experimental_Astro.Container.addServerRenderer`.
+
+In the following example, we manually import the server renderers for React and Vue components, and MDX for `.mdx` pages. 
+
+```js
+import reactRenderer from "@astrojs/react/server.js";
+import vueRenderer from "@astrojs/vue/server.js";
+import mdxRenderer from "astro/jsx/server.js";
+
+const container = await experimental_AstroContainer.create();
```

Comment:
@ematipico I wasn't sure whether we still also need the `experimental` prefix here and in the text in line 89? I added it because the other example does use it, but you can remove it if it's not correct.

Reply:
Oh yeah, that sounds good